lib/rucio/rse/protocols/protocol.py

```diff
@@ -143,7 +143,13 @@ def _module_init_(cls):
         except (NoOptionError, NoSectionError):
             pass
         if policy_module:
-            importlib.import_module(policy_module)
+            # TODO: The import of importlib is done like this due to a depdency issue with python 2.6 and incompatibility of the module with py3.x
+            # More information https://github.com/rucio/rucio/issues/875
+            try:
+                import importlib
+                importlib.import_module(policy_module)
+            except:
+                pass
```

Comment:
should it fail silently ?

Reply:
Right, it should just raise it through...